tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/ExpectedAVAudioPcmBufferDefaultCtr.cs

```diff
@@ -0,0 +1,46 @@
+// <auto-generated />
+
+#nullable enable
+
+using Foundation;
+using ObjCBindings;
+using ObjCRuntime;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+
+namespace TestNamespace;
+
+[Register ("AVAudioPCMBuffer", true)]
+public partial class AVAudioPcmBuffer
+{
+	[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+	static readonly NativeHandle class_ptr = Class.GetHandle ("AVAudioPCMBuffer");
+
+	public override NativeHandle ClassHandle => class_ptr;
+
+	[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+	[EditorBrowsable (EditorBrowsableState.Never)]
```

Comment:
This EditorBrowsable attribute doesn't look right.

Reply:
Should be advance if I look at been. I'll fix that.